CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Ewon Flexy Extensions Library Changelog
```

Comment:
Commit name is improper: `RELEASE: v.14.2`

Should be `RELEASE: v1.14.2`

Reply:
done